src/styles/_catalog.scss

```diff
@@ -1,9 +1,12 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
let's create variable for main size (200px)

Reply:
Done.